src/App.js

```diff
@@ -7,30 +7,34 @@ const { default: CreateMuiTheme } = require('@material-ui/core/styles/createMuiT
 const { default: MuiThemeProvider } = require('@material-ui/styles/ThemeProvider');
 const { default: CssBaseline } = require('@material-ui/core/CssBaseline');
 const StrangeRouter = require('strange-router');
+const MiddleEnd = require('strange-middle-end');
 const { ConnectedRouter } = require('connected-react-router');
 const { default: ErrorBoundary } = require('react-error-boundary');
 const ErrorFallback = require('./components/ErrorFallback');
 const Routes = require('./routes');
 
-module.exports = ({ store, theme = CreateMuiTheme(), Router = ConnectedRouter, ...routerProps }) => {
+module.exports = ({ middleEnd, store, theme = CreateMuiTheme(), Router = ConnectedRouter, ...routerProps }) => {
```

Comment:
You can omit the store here, and just reference `middleEnd.store` below.

Reply:
Neat!